docs/source/basic_setup.rst

```diff
@@ -73,6 +87,22 @@ create the directory for the submission files anywhere, just note the location (
     $ mkdir ~/Documents/submit_cgap
 
 
+Installing XCode Developer Tools
```

Comment:
These can also be installed from the Apple Store, but _probably_ we don't need to mention that. It's how I do it.